.github/workflows/build_latest.yml

```diff
@@ -20,7 +20,7 @@ jobs:
       fail-fast: false
       matrix:
         os: ['ubuntu', 'windows']
-        qt-version: [ '5.12.*', '5.15.*', '6.7.*' ]
+        qt-version: [ '5.12.*', '5.15.*', '6.7.*', '6.8.*' ]
```

Comment:
`aqtinstall` only advertises support for 6.8.1, not 6.8.2, so maybe this has to be pinned for the time being

Reply:
Ok, that was not correct - actually the architecture has changed for ubuntu with Qt >= 6.7 (`linux_gcc_64` instead of `gcc_64`) and for Windows with Qt 6.8 (`win64_msvc2022_64`). Though now the wrapper generation fails...

Reply:
The problem seems to be the macro `Q_DECLARE_STRONGLY_ORDERED` defined in _qtscript_masterinclude.h_. Should probably be adapted for Qt 6.8...